jetson/teleop/src/__main__.py

```diff
@@ -129,6 +130,11 @@ def drive_control_callback(channel, msg):
 
         joystick_math(new_motor, magnitude, theta)
 
+        if(reverse):

```

Comment:
why is there a reverse check in kill()?

Reply:
I don't think there is? Github may just be hiding it but this is in drive_control_callback